dockerfiles/predict_model.dockerfile

```diff
@@ -5,13 +5,16 @@ RUN apt update && \
     apt install --no-install-recommends -y build-essential gcc && \
     apt clean && rm -rf /var/lib/apt/lists/*
 
+#We install requirements before any copying for better caching 
 COPY requirements.txt requirements.txt
-COPY pyproject.toml pyproject.toml
```

Comment:
Is this not needed?

Reply:
It works without it. Maybe we need it later for something!?! We can add it back if needed later or I can add it back now I got no preferences.

Reply:
you decide :PP

Reply:
I know make commands fail if smth is wrong or missing in pyproject.toml, confirmed this yesterday hehe

Reply:
Aight then I am putting it back in :)